clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -183,6 +192,38 @@ func (pc *PrometheusController) exposeKubemarkApiServerMetrics() error {
 	return nil
 }
 
+// runKubemarkNodeExporter creates daemonset with node-exporter in kubemark (not "root"!) cluster.
+func (pc *PrometheusController) runKubemarkNodeExporter() error {
+	klog.Infof("Starting node-exporter on kubemark's master nodes.")
+	kubemarkFramework, err := framework.NewFramework(&pc.clusterLoaderConfig.ClusterConfig, numK8sClients)
+	if err != nil {
+		return err
+	}
+
+	// Validate masters first
+	nodes, err := client.ListNodes(kubemarkFramework.GetClientSets().GetClient())
+	if err != nil {
+		return err
+	}
+
+	numMasters := 0
+	for _, node := range nodes {
+		if system.IsMasterNode(node.Name) {
+			numMasters++
+			if label := node.Labels[nodeExporterNodeLabel]; label != "true" {
+				return fmt.Errorf("Master %q doesn't have required %q label which is required by node-expoter", node.Name, nodeExporterNodeLabel)
```

Comment:
NIT: error strings in go shouldn't start with capital letter
NIT: s/node-expoter/node-exporter/